### PR TITLE
test setup for finch

### LINKED ISSUE #12: Unit test setup
Setup Unit testing using jest & write test for all the current modules written in application

### PR DESCRIPTION
**Pull request by anshulsahni**
Fixes https://github.com/anshulsahni/easy-raseed/issues/12

### CODE REVIEW

**Comment by anshulsahni** [comment]
@tarunbatra added test for all the elements of a Books module, please check my strategy is good or not?

File: finch/src/modules/base/__tests__/crud-controller.test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      const mockRes = httpMocks.createResponse();
+
+      const mockService = {
+        getById: jest.fn().mockReturnValueOnce({
+          sample: 'response',
+        }),
+      };
+
+      const controller = new CrudController({});
+      controller.service = mockService;
```
Comment (tarunbatra) [inline finch/src/modules/base/__tests__/crud-controller.test.js]: might want to do this in a `beforeEach`.  All the methods in `mockService` can be defined in it and later in test we can just override the mocks.

**Review by tarunbatra — APPROVED** [review]
Few nitpicks aside, LGTM

**Comment by anshulsahni** [comment]
@tarunbatra done changes as suggested by you, please check once

**Pull request merged**